sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Grpc/ServiceBusGrpcSessionEndToEndTests.cs

```diff
@@ -140,6 +145,67 @@ public async Task BindToSessionMessageAndAbandon()
             Assert.IsEmpty(provider.ActionsCache);
         }
 
+        [Test]
+        public async Task BindToSessionMessageAndSetAndGet()
+        {
+            var host = BuildHost<ServiceBusBindToSessionMessageAndSetAndGet>();
+            var settlementImpl = host.Services.GetRequiredService<SettlementService>();
+            var provider = host.Services.GetRequiredService<MessagingProvider>();
+            ServiceBusBindToSessionMessageAndSetAndGet.SettlementService = settlementImpl;
+            await using ServiceBusClient client = new ServiceBusClient(ServiceBusTestEnvironment.Instance.ServiceBusConnectionString);
+
+            using (host)
+            {
+                var message = new ServiceBusMessage("foobar") { SessionId = "sessionId" };
```

Comment:
Can we add a test case with binary data rather than string?

Reply:
Added!